Tests/Command/GeocodeCommandTest.php

```diff
@@ -44,26 +46,24 @@ public function testExecute()
             'countryCode' => $country->getCode(),
         ]);
 
-        $geocoder = $this->getMockBuilder(ProviderAggregator::class)->getMock();
+        $geocoder = $this->createMock(ProviderAggregator::class);
```

Comment:
Are you sure about this change?

Wasn't `createMock` deprecated it PHPUnit6?

Reply:
It does exist in 8.2